code/modules/admin/verbs/massmodvar.dm

```diff
@@ -453,6 +453,38 @@
 						if (A.type == O.type)
 							A.vars[variable] = O.vars[variable]
 
+	if(method)
+		if(istype(O,/mob))
+			for(var/mob/M in mob_list)
+				if(istype(M,O.type))
+					M.on_varedit(variable)
+
+		else if(istype(O,/obj))
+			for(var/obj/A in world)
+				if(istype(A,O.type))
+					A.on_varedit(variable)
+
+		else if(istype(O,/turf))
+			for(var/turf/A in world)
+				if(istype(A,O.type))
+					A.on_varedit(variable)

```

Comment:
Doesn't BYOND handle this type of loop in a special way, though?  Or maybe I'm thinking of `for(var/turf/T)`, without the `in world`.  Guess it's worth testing.